app/services/api/axios.ts

```diff
@@ -12,28 +13,29 @@ export type PaginatedItems<T> = PaginatedData & {
 }
 
 export const axiosInstance = axios.create({
-  baseURL: "https://chain-react-ai-chat.vercel.app/api/schedule/",
-  // baseURL: "http://localhost:3000/api/schedule/",
+  baseURL: WEBFLOW_API_URL,
   headers: {
     "Content-Type": "application/json",
     "User-Agent": "Webflow Javascript SDK / 1.0",
   },
 })
 
-axiosInstance.interceptors.response.use(
-  (response) => {
-    return response
-  },
-  (error) => {
-    console.tron.log({ error })
-    return Promise.reject(error)
-  },
-)
+if (typeof __DEV__ !== "undefined" && __DEV__ === true) {
```

Comment:
Hiding `__DEV__` behind guard so we can consume this file in `scripts/webflow-codegen.ts`, which doesn't use Metro